### PR TITLE
Update tfa support for clearscore

### PR DESCRIPTION
**Pull request by poisonadder1**
Clearscore has added support for sms based 2fa.

### CODE REVIEW

File: _data/finance.yml [hunk 1]
```diff
@@ -62,9 +62,10 @@ websites:
 
   - name: ClearScore
     url: https://www.clearscore.com
-    twitter: ClearScore
-    facebook: clearscore
     img: clearscore.png
+    tfa:
+      - sms
+    doc: https://help.clearscore.com/hc/en-us/articles/360010189440-About-two-factor-authentication
```
Comment (Carlgo11) [inline _data/finance.yml]:
```suggestion
    doc: https://help.clearscore.com/hc/en-us/sections/360003360940
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
The documentation doesn't mention what kind of 2FA they support. Are you sure they only use SMS based 2FA?

**Comment by poisonadder1** [comment]
> The documentation doesn't mention what kind of 2FA they support. Are you sure they only use SMS based 2FA?

As far as I've been able to tell.

**Review by Jawshy — APPROVED** [review]
@Carlgo11 see [What happens after I’ve enabled Two-Factor Authentication (2FA)?](https://help.clearscore.com/hc/en-us/articles/360011124640-What-happens-after-I-ve-enabled-Two-Factor-Authentication-2FA-) 

> If you log in on a new device (or not actively logged in on your app), you’ll need to enter your email and password and the verification code will be sent to you via text as you’ve enabled 2FA.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**